doc/CMakeLists.txt

```diff
@@ -13,11 +13,17 @@ if(DOXYGEN_FOUND)
         add_custom_target(doc)
     endif()
 
+    add_custom_target(cpp-apidoc-content-tags
```

Comment:
Fixed, though requires `&&` to not append doxygen command as an argument to script.
Not certain if you prefer these arguments as specific cmake variables, so I simply appended for now.